docs/index.html

```diff
@@ -90,6 +90,7 @@ <h2 class="title">A 1x Engineer...</h2>
           <li>May decorate their laptop and workspace in any way they like, and is respectful of others' decor (or lack thereof), too.</li>
           <li>Isn't defined by myopic Tweetstorms by clueless Venture Capitalists (VCs).</li>
           <li>Doesn't ridicule entire professions within engineering, especially not when in a position of leadership.</li>
+          <li>Doesn't look down on teammates despite his status and projects achievements; But help them get good with humility.</li> 
```

Comment:
Let's make it gender-neutral while we're at it :)

Reply:
Oh... Right.